driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -165,6 +165,19 @@ public static Bson out(final String collectionName) {
         return new BsonDocument("$out", new BsonString(collectionName));
     }
 
+    /**
+     * Creates a $sample pipeline stage with the specified sample size
+     *
+     * @param size the sample size
+     * @return the $sample pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/sample/  $sample
+     * @mongodb.server.release 3.2
+     * @since 3.2
+     */
+    public static Bson sample(final int size) {
+        return new BsonDocument("$sample", new BsonDocument("size", new BsonInt32(size)));
```

Comment:
Its really just to create the stage - so its not coupled to other stages.

Any information about disk use? http://docs.mongodb.org/manual/release-notes/3.1-dev-series-reference/#pipe._S_sample seems lacking on the subject - although the eventual manual docs should cover any caveats, which should be enough.